parakeet/browser.py

```diff
@@ -230,13 +247,28 @@ def retry(self, method=None, **kwargs):
 
             return result
         except Exception as ex:
-            LOG.error('Exception: {}'.format(ex.message))
+            LOG.error('Exception: {}'.format(str(ex)))
```

Comment:
Add a better log message. I'd suggest it to be WARN, as you have some handling of exceptional cases.
I would add an error message when the attempts to retry are over (at the end of the 'except').